core/src/test/java/nu/yona/server/test/util/JUnitUtil.java

```diff
@@ -65,4 +71,43 @@ public Object answer(InvocationOnMock invocation) throws Throwable
 		RepositoryProvider.setRepositories(mockRepositories);
 		return mockRepositories;
 	}
+
+	public static User createRichard()
+	{
+		return createUserEntity("Richard", "Quinn", "+31610000000", "RQ", "topSecret");
+	}
+
+	public static User createBob()
+	{
+		return createUserEntity("Bob", "Dunn", "+31610000001", "BD", "heelGeheim");
+	}
+
+	public static User createUserEntity(String firstName, String lastName, String mobileNumber, String nickname,
+			String vpnPassword)
+	{
+		MessageSource anonymousMessageSource = MessageSource.createInstance();
+		UserAnonymized johnAnonymized = UserAnonymized.createInstance(anonymousMessageSource.getDestination(),
+				Collections.emptySet());
+		UserAnonymized.getRepository().save(johnAnonymized);
+
+		byte[] initializationVector = CryptoSession.getCurrent().generateInitializationVector();
+		MessageSource namedMessageSource = MessageSource.createInstance();
+		UserPrivate userPrivate = UserPrivate.createInstance(nickname, vpnPassword, johnAnonymized.getId(),
+				anonymousMessageSource.getId(), namedMessageSource);
+		return new User(UUID.randomUUID(), initializationVector, firstName, lastName, mobileNumber, userPrivate,
```

Comment:
Shouldn't this be saved in the `UserRepository` too?

Reply:
OK, will do.